h/presenters.py

```diff
@@ -33,13 +35,13 @@ def truncate(content, length=60):
     link_text = truncate(link_text)
 
     if href and hostname:
-        link = '<a href="{href}" title="{title}">{link_text}</a><br>({hostname})'
-    elif hostname:
-        link = '<a title="{title}">{link_text}</a><br>({hostname})'
+        link = '<a href="{href}" title="{title}">{link_text}</a><br>{hostname}'
     elif href:
         link = '<a href="{href}" title="{title}">{link_text}</a>'
     else:
-        link = '<a title="{title}">{link_text}</a>'
+        link = '<em>Local file:</em> {title}'
+        if hostname:
+            link += '<br>{hostname}'
```

Comment:
This is showing the `Local file` prefix for all documents where the link is not http or https, which is technically wrong, but leaving the old `a` tag without an `href` and branching off again didn't feel like the right way either. Any other thoughts on this?